Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py

```diff
@@ -21,34 +20,38 @@
 """ Types """
 FetchIncidentsStorage = TypedDict("FetchIncidentsStorage", {
     "last_modified_fetched": str,
-    "last_modified_offset": str,
     "zf-ids": list[str],
 })
 
 
 """ CLIENT """
 
+ALLOWED_SORT_FIELDS = ["timestamp", "last_modified"]
+ALLOWED_ALERT_FILTERS = ["last_modified_min_date",
+                         "min_timestamp", "max_timestamp"]
+
 
 class ZFClient(BaseClient):
     def __init__(
-        self, username, password, fetch_limit, only_escalated, *args, **kwargs
+        self, username, password, only_escalated, *args, **kwargs
     ):
         super().__init__(*args, **kwargs)
         self.credentials = {
             "username": username,
             "password": password
         }
-        self.fetch_limit = fetch_limit
```

Comment:
Why did you remove the fetch_limit? 
I understand it fetches now all alerts, but if there is a big number of alerts, thousands for example, it might cause performance issues.

Reply:
fetch limit was causing issues with getting incidents and then matching timestamps with the get_modified_remote_data  command, therefore we decided to delete it
alerts don't show in the thousands so far as we've researched, at least if incidents aren't fetched at least hourly

Reply:
I understand, I still think that the best practice here is to support it too in order to avoid breaking changes and future bugs as we don't know how the customers are using the fetch mechanism. 
But I leave this decision up to you.

Reply:
I believe the biggest problem lies on the first time fetch_incidents runs, but that is something we we'll improve on in the future, we would still like for this change to be included